src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/CollectionsShouldImplementGenericInterfaceTests.cs

```diff
@@ -12,12 +12,366 @@ public class CollectionsShouldImplementGenericInterfaceTests : DiagnosticAnalyze
     {
         protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
         {
-            return new BasicCollectionsShouldImplementGenericInterfaceAnalyzer();
+            return new CollectionsShouldImplementGenericInterfaceAnalyzer();
         }
 
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
-            return new CSharpCollectionsShouldImplementGenericInterfaceAnalyzer();
+            return new CollectionsShouldImplementGenericInterfaceAnalyzer();
+        }
+
+        [Fact]
+        public void Test_WithCollectionBase()
+        {
+            #region CSharp Test
+            VerifyCSharp(@"
+                        using System;
+                        using System.Collections;
+                        using System.Collections.Generic;
```

Comment:
Just curious why your test code mentions `System.Collections.Generic`, even though it doesn't reference anything from that namespace.